opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -500,19 +500,14 @@ message DoubleHistogramDataPoint {
   // TODO: Add more bucket options.
 
   // explicit_bounds specifies buckets with explicitly defined bounds for values.
-  // The bucket boundaries are described by "bounds" field.
   //
-  // This defines size(bounds) + 1 (= N) buckets. The boundaries for bucket
-  // at index i are:
+  // This defines size(explicit_bounds) + 1 (= N) buckets. The boundaries for
+  // bucket at index i are:
```

Comment:
This looks right, but the use of `()` and `[]` for open- and closed-intervals makes it difficult to see the change. I'd like to add something like:

```
Histogram buckets are inclusive of their upper boundary, except the last bucket where the boundary is at infinity. This format is intentionally compatible with the OpenMetrics histogram definition.
```

Reply:
Done.